README.md

```diff
@@ -87,6 +87,14 @@ INSERT INTO "schema_migrations" (version) VALUES
 ;
 ```
 
+By default the gem will remove some extensions that typically aren't needed in structure dumps. You can choose to keep all extensions:
+
+```ruby
+Rails.application.configure do
+  config.activerecord_clean_db_structure.do_not_remove_extensions = true
```

Comment:
I wonder if we should instead call this something like:

```ruby
   config.activerecord_clean_db_structure.keep_extensions = :all
```

That way in a future iteration of this we can more easily transition it to a specific allowlist of which extensions to keep (which would make this an array of strings in those cases).

Reply:
@ghiculescu I love the super-clear name of the setting, but I also see the point @lfittl is making.

Expanding on that: the implementation in the future could grab this setting value, split on commas, stringify, and if "all" appears anywhere in the list or first, all extensions are kept. Otherwise, it would be an explicit list of extension names to keep.  That way it would be backward compatible for newer versions of this gem and older configurations.

Reply:
@ghiculescu If you agree with ^^^ and are willing to update the implementation to support that, that would be great. We can include this in the next gem release.

Reply:
Okay, I pushed a new version :)